FireRoad/CourseBrowserCell.swift

```diff
@@ -55,7 +61,12 @@ class CourseBrowserCell: UITableViewCell {
     var course: Course? = nil {
         didSet {
             updateTitle()
+            
             self.descriptionLabel.text = course?.subjectTitle
```

Comment:
Better to create a temporary variable for the description label text and only set the label's text at the end (similar to `updateTitle`)